src/main/java/com/google/devtools/common/options/OptionsClassProvider.java

```diff
@@ -19,6 +19,7 @@
  * A read-only interface for options parser results, which only allows to query the options of
  * a specific class, but not e.g. the residue any other information pertaining to the command line.
  */
+@FunctionalInterface
```

Comment:
@dslomov I made a reasonable guess that it was meant to effectively be a functional interface, because it currently has only one non-default method.

Is it likely for `OptionsClassProvider` to get new non-default methods in the future?